### ISSUE TITLE
[Docs] Group migration and edge channel docs

### ISSUE THREAD

**Issue by clemcode** [nuxt/framework#7716]
Currently, the migration guides summary and the edge channel page are grouped in two separate pages under the "Upgrade guide" section:

- https://v3.nuxtjs.org/getting-started/upgrade-guide/migration
- https://v3.nuxtjs.org/getting-started/upgrade-guide/edge-channel

These pages could be merged onto a single "Upgrade guide" page with the following structure:

- (Intro)
- Comparison table
- From Nuxt 2 to Nuxt 3
- From Nuxt 2 to Bridge
- Edge Release Channel

This would lead to a more flat structure and group all upgrade informations in the same place to make them more discoverable.

**Comment by danielroe** [comment]
This sounds good to me 👍

**Comment by pi0** [comment]
Agreed that having two categories in getting started is not nice.  

For edge: We might move edge channel to the guide>advanced. (https://github.com/nuxt/framework/issues/7376) it is not suitable for starters really nor for normal users as we are moving toward 3.0.0 there should be fewer reasons to advocate edge. 

If the upgrade guide were entirely inside the guide, we could append edge channel to the end of it but it is in getting started right now...

For Nuxt 2: migration from Nuxt 2 to Bridge/Nuxt3 shall be added to the end of getting started. Main audience of v3 docs as Nuxt 3 users we are writing upgrade guide for them.

---

Getting started > Upgrade Guide:
 - Upgrade Nuxt 3
 - Upgrade from Nuxt 2
    -  Comparison table
    -  Bridge
    -  Nuxt 3
Guide > Going Further
    - Edge Channel
